exams/management/commands/import_edx_exam_grades.py

```diff
@@ -0,0 +1,81 @@
+"""
+Import proctored exam grades from edx
+"""
+import csv
+import argparse
+from django.contrib.auth.models import User
+
+from django.core.management import BaseCommand, CommandError
+
+from courses.models import Course
+from exams.models import ExamRun, ExamAuthorization
+from exams.pearson.constants import EXAM_GRADE_PASS, EXAM_GRADE_FAIL
+from grades.models import ProctoredExamGrade
+
+
+class Command(BaseCommand):
+    """Parses a csv with exam grades creating or updating ProctoredExamGrade"""
+    help = "Parses a csv with exam grades and creates ProctoredExamGrade"
+
+    def add_arguments(self, parser):
+        parser.add_argument('csvfile', type=argparse.FileType('r'), help='')
+
+        parser.add_argument('course_code', type=argparse.FileType('r'), help='Example: 14.100')
+
+    def handle(self, *args, **kwargs):  # pylint: disable=unused-argument,too-many-locals
+
+        csvfile = kwargs.get('csvfile')
+        course_number = kwargs.get('course_code')
+        reader = csv.DictReader(csvfile.read().splitlines())
+
+        try:
+            course = Course.objects.get(course_number__startswith=course_number)
+        except Course.DoesNotExist:
+            raise CommandError(
+                'Could not find a course with number "{}"'.format(course_number)
+            )
+        except Course.MultipleObjectsReturned:
+            raise CommandError(
+                'There are multiple courses with given number "{}"'.format(course_number)
+            )
+        # should be only one current exam run
+        exam_run = ExamRun.get_currently_schedulable(course).first()
```

Comment:
Can we use `except ExamRun.DoesNotExist` here, and also `ExamRun.MultipleObjectsReturned` to detect if there are more than one matching `ExamRun`?